cypress/integration/tests/controls/stakeholdergroups/create.test.ts

```diff
@@ -30,6 +32,36 @@ describe("A single Stakeholder group", () => {
         cy.wait("@getStakeholdergroups");
 
         // Assert that newly created stakeholder group is deleted
-        cy.get(tdTag).should("not.contain", stakeholdergroup.stakeholdergroupName);
+        cy.get(tdTag).should("not.contain", stakeholdergroup.getCurrentstakeholdergroupName);
+    });
+
+    it("Stakeholder group CRUD with Stakeholder Member attached", function () {
+        // Create stakeholder
+        stakeholders.create();
+        var stakeholderName;
+        stakeholderName = stakeholders.getCurrentStakeholderName();
+        // Create new stakeholder group
+        stakeholdergroup.create(stakeholderName);
+        cy.wait("@postStakeholdergroups");
+        // Check if stakeholder member attached to stakeholder group
+        cy.get(tdTag).should("contain", "1");
+
+        // Edit stakeholder group with name, description and member
+        stakeholdergroup.edit(stakeholderName);
```

Comment:
If we are not removing the member , how will the count reduce to 0 ?

Reply:
Actually, this the PF4 dropdown widget behaviour where if we select same item which is already selected then it removes element.